release-notes/opensearch-release-notes-1.3.7.md

```diff
@@ -0,0 +1,71 @@
+# OpenSearch and OpenSearch Dashboards 1.3.7 Release Notes
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 1.3.7 includes the following bug fixes, infrastructure and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.7.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.7.md).
+
```

Comment:
Add new section header "HIGHLIGHTS"

Add new subhead "Windows Support"

Add description: 
This release introduces Windows x64 distributions for OpenSearch and OpenSearch Dashboards for the 1.3 line. Available in ZIP format, the distributions allow users to deploy OpenSearch 1.3.7 directly in their Windows environment.

Reply:
@rishabh6788 let's use `Release Highlights` as in this release notes for the header.
https://raw.githubusercontent.com/opensearch-project/opensearch-build/main/release-notes/opensearch-release-notes-2.4.0.md